modules/n1ql/pages/n1ql-language-reference/groupby.adoc

```diff
@@ -49,6 +50,51 @@ For details, see xref:n1ql-language-reference/from.adoc#section_ax5_2nx_1db[AS C
 Assigning an alias to the group term is optional.
 If you assign an alias, the `AS` keyword may be omitted.
 
+[[group-as]]
+=== GROUP AS Clause
+
+[source,ebnf]
+----
+include::partial$grammar/dql.ebnf[tag=group-as-clause]
+----
+
+image::n1ql-language-reference/group-as-clause.png["Syntax diagram", align=left]
+
+JSON is a hierarchical format, and a fully featured JSON query language needs to be able to produce hierarchies of its own, with computed data at every level of the hierarchy. 
+The key feature of SQL{pp} that makes this possible is the `GROUP AS` clause.
+
+A query can include a `GROUP AS` clause only if it has a `GROUP BY` clause. 
+The effect of a `GROUP BY` clause is to hide the original objects in each group, exposing only the grouping expressions and special aggregation functions on the non-grouping fields. 
+The purpose of the `GROUP AS` clause is to make the original objects in the group visible to subsequent clauses. 
+As a result, the query can generate output data both for the group as a whole and for the individual objects inside the group.
+
+For each group, `GROUP AS` preserves all of the objects in the group, as they were before grouping, in an array. 
+The name of the array is the alias in the GROUP AS clause. 
+You can then use the array name in the `FROM` clause of a subquery to process and return the individual objects in the group.
+
+If the `FROM` clause of the query includes an alias or aliases, the `GROUP AS` array wraps each object in an outer object identified by its `FROM` alias. 
```

Comment:
Please change this line "If the `FROM` clause of the query includes an alias or aliases, the `GROUP AS` array wraps each object in an outer object identified by its `FROM` alias. "

... to something along the lines of - 

In the GROUP AS array, each object in the group is wrapped in an outer object that gives it the name of the alias if specified or the implicit alias of its corresponding from-term in the FROM clause.

[ The reason I ask for this change is because the FROM clause need not explicitly have an alias mentioned.  If no alias is set example if simply like - SELECT * FROM airline; the alias of the "airline" from-term is simply "airline". This is the implicit alias ]

Reply:
ah, I see it now, thank you